src/Controls/src/Core/HandlerImpl/RadioButton/RadioButton.Tizen.cs

```diff
@@ -104,12 +108,24 @@ static View BuildTizenDefaultTemplate()
 
 			VisualState checkedVisualState = new VisualState() { Name = CheckedVisualState };
 			checkedVisualState.Setters.Add(new Setter() { Property = OpacityProperty, TargetName = CheckedIndicator, Value = 1 });
-			checkedVisualState.Setters.Add(new Setter() { Property = Shape.StrokeProperty, TargetName = UncheckedButton, Value = RadioButtonCheckMarkThemeColor });
+			checkedVisualState.Setters.Add(
+				new Setter()
+				{
+					Property = Shape.StrokeProperty,
+					TargetName = UncheckedButton,
+					Value = new AppThemeBinding() { Light = Colors.Black.MultiplyAlpha(0.1f), Dark = Colors.White.MultiplyAlpha(0.1f) }
```

Comment:
@PureWeen i see in shell you got hardcoded black and white too. Are theme resources not linkable in c#? Will this break people now since the theme resource is not in use?

Reply:
Hmm, I think we'd ideally have Theme Dictionaries here that we could key off of for `Light` and `Dark`. I don't think there's any easy way for users to articulate a different `Light` and `Dark` color themselves here. 

 With the absence of Theme Dictionaries I'm thinking maybe one of these options? 
 
 1) We create different colors suffixed with "light" and "dark" so users can specify the different ones. So we'll have
 - RadioButtonThemeColor
 - RadioButtonThemeColorLight
 - RadioButtonThemeColorDark

2) Can we use `DynamicResources` for the properties here if we detect that users have set a key?  Ideally we'd have some default "keys" set at the application level but for now we can just check if the thing is set. 

Something like this seems to work (code needs to be cleaned up a bit)

```C#
if (Application.Current.TryGetResource("RadioButtonThemeColor", out _))
	normalEllipse.SetDynamicResource(Ellipse.StrokeProperty, "RadioButtonThemeColor");
else
	normalEllipse.SetAppTheme(Ellipse.StrokeProperty, SolidColorBrush.Black, SolidColorBrush.White);
```

And then on the VSM kind of the same thing
```C#
object setterValue;
			if (Application.Current.TryGetResource("RadioButtonThemeColor", out _))
				setterValue = new DynamicResource("RadioButtonCheckMarkThemeColor");
			else
				setterValue = new AppThemeBinding() { Dark = SolidColorBrush.White, Light = SolidColorBrush.Black };


			checkedVisualState.Setters.Add(
				new Setter()
				{
					Property = Shape.StrokeProperty,
					TargetName = UncheckedButton,
					Value = setterValue
				});
```

Reply:
Can we have the light and dark dynamic resources, and then have an app theme binding to both? Not sure if that is supported.

Also, a nit on the name: RadioButtonCheckGlyphStroke since the property is a brush I think?Git the name from https://github.com/microsoft/microsoft-ui-xaml/blob/4283cb89e0c8b63ac011e8d46a81a79b6accdc45/dev/CommonStyles/RadioButton_themeresources.xaml#L46

I see WinUI has theme dictionaries, but we have theme bindings. So maybe we have the mix of option 1 and 2. We have 2 dynamic resources `RadioButtonCheckGlyphStrokeLight` and `RadioButtonCheckGlyphStrokeDark` and then have a theme binding to those? Then we can set those colors in our templates. If those properties do not exist, we can just use black/white we always did.

Not sure if the theme binding can bind to dynamic resources...

Reply:
Seems you can't bind a "dynamic theme resource". 😭

Reply:
*NOTES:*
* The current logic in `ResolveThemeColor` is not super sensible as it just gets the color and then falls back to white/black without regard to what it is used for. 
* I also see the names are bad since they say color but are a brush... but...

I am somehow coming to a point of a mix of these options. I like the XxxDark and XxxLight values in the dictionaries just so they are there and easy to change, and they also show that it is not dynamic - it is set once and we use another mechanism for the dynamic - AppThemeBinding.

But the counter to that is that we have the non-XxxDark/Light value for compatibility.

> It is a tiny bit unfortunate that we don't have the AppThemeBinding to a DynamicResource, but that is a separate feature.

My final-ish thoughts is that we can do:

* Add the `RadioButtonCheckMarkThemeColorLight` and `RadioButtonCheckMarkThemeColorDark` for a static resource in the templates
* Only use the `RadioButtonCheckMarkThemeColor` for a dynamic resource as compatibility
* Use `AppThemeBinding` in the properties to the XxxDark and XxxLight (IF they exist)
* Fall back to DynamicResource (IF Xxx exists)
* Fall back to hardcoded black/white if nothing exists

The benefit of using the app theme is that it changes with the OS and the benefit of the light/dark is that they can be overridden separately AND without having to update the radio button control template.

This does mean that the value of XxxDark cannot change at runtime once the radio button is created, but that is just what we have until we make AppThemeBindings support dynamic. And, we are not losing anything as this value was static from app startup/first radio button creation.

The code version of this would be:

```cs
// first try new light/dark keys
if (Application.Current.TryGetResource("RadioButtonOuterEllipseStrokeLight", out var outerLight) && 
    Application.Current.TryGetResource("RadioButtonOuterEllipseStrokeDark", out var outerDark))
	normalEllipse.SetAppTheme(Ellipse.StrokeProperty, outerLight, outerDark);
// try older resource key
else if (Application.Current.TryGetResource("RadioButtonThemeColor", out _))
	normalEllipse.SetDynamicResource(Ellipse.StrokeProperty, "RadioButtonThemeColor");
// just use black/white
else
	normalEllipse.SetAppTheme(Ellipse.StrokeProperty, SolidColorBrush.Black, SolidColorBrush.White);
```

Using the resource names from WinUI:

* RadioButtonOuterEllipseStroke
* RadioButtonCheckGlyphStroke

Reply:
@PureWeen 😭 sorry, my brain dump here... I edited this comment like 10 times before posting with all different things.